src/libsync/vfs/cfapi/cfapiwrapper.h

```diff
@@ -82,18 +82,18 @@ NEXTCLOUD_CFAPI_EXPORT bool isSparseFile(const QString &path);
 
 NEXTCLOUD_CFAPI_EXPORT FileHandle handleForPath(const QString &path);
 
-PlaceHolderInfo findPlaceholderInfo(const FileHandle &handle);
+PlaceHolderInfo findPlaceholderInfo(const QString & path);
```

Comment:
Please remove the space after `&`, so, `QString & path` becomes `QString &path`.